core/commands/add.go

```diff
@@ -30,9 +30,11 @@ const (
 
 var AddCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline: "Add a file or directory to ipfs.",
+		Tagline: "Add a file to ipfs.",
```

Comment:
@RichardLitt Out of curiosity, is there a reason for this change.  The original text seamed more accurate.